server/migrations/1681833645009_create-user-migration-table.js

```diff
@@ -0,0 +1,44 @@
+import { PromisePool } from '@supercharge/promise-pool';
+import keycloak from '../keycloak';
+import { collections } from '../db';
+
+exports.shorthands = undefined;
+
+exports.up = async (pgm) => {
+  const users = [];
+
+  const keycloakUsers = await keycloak.getUsers();
```

Comment:
Isn't an issue here that this will grab users from the target env?
I think let's keep the migration to just the table creation and populate it separately so as to avoid adding extra ENVs for the source keycloak environment.

Reply:
This migration should be run with the current keycloak and we don't need additional envs.

Reply:
I don't think we can separate that.

Reply:
Now, I get what you mean.